apis/externalsecrets/v1beta1/secretstore_validator.go

```diff
@@ -54,9 +55,26 @@ func (r *GenericStoreValidator) ValidateDelete(_ context.Context, _ runtime.Obje
 }
 
 func validateStore(store GenericStore) (admission.Warnings, error) {
+
+	if len(store.GetSpec().Conditions) != 0 {
+		validateConditions(store)
+	}
 	provider, err := GetProvider(store)
 	if err != nil {
 		return nil, err
 	}
 	return nil, provider.ValidateStore(store)
 }
+
+func validateConditions(store GenericStore) error {
+	for _, condition := range store.GetSpec().Conditions {
+		if len(condition.NamespacesRegex) != 0 {
```

Comment:
nit, we don't have to check the length here since if `condition.NamespacesRegex` is empty or nil, the for loop does not run anyway 🙂